sigh/lib/assets/resign.sh

```diff
@@ -499,43 +515,13 @@ function resign {
     # Replace embedded provisioning profile with new file
     cp -f "$NEW_PROVISION" "$APP_PATH/embedded.mobileprovision"
 
-    #if the current bundle identifier is different from the new one in the provisioning profile, then change it.
```

Comment:
could we keep this (and the follow 2) comments so it is clear what these new `change_plist_value` lines do?